app-office/calligra/calligra-5.9999.ebuild

```diff
@@ -0,0 +1,244 @@
+	sys-libs/zlib
+	colorio? ( media-libs/opencolorio )
+	crypt? ( >=app-crypt/qca-2.1.0:2[qt5] )
+	dbus? ( dev-qt/qtdbus:5 )
+	diagrams? ( $(add_kdeapps_dep kdiagram) )
+	exif? ( >=media-gfx/exiv2-0.16:= )
+	fontconfig? ( media-libs/fontconfig )
+	gsl? ( sci-libs/gsl )
+	import-filter? (
+		app-text/libetonyek
+		app-text/libodfgen
+		app-text/libwpd:0.10
+		app-text/libwpg:0.3
+		app-text/libwps
+		dev-libs/librevenge
+		media-libs/libvisio
+	)
+	jpeg2k? ( media-libs/openjpeg:0 )
+	lcms? ( >=media-libs/lcms-2.4:2 )
+	openexr? ( media-libs/openexr )
+	opengl? ( virtual/opengl )
+	okular? ( $(add_kdeapps_dep okular) )
+	pdf? ( >=app-text/poppler-0.28:=[qt5] )
+	phonon? ( media-libs/phonon[qt5] )
+	raw? ( $(add_kdeapps_dep libkdcraw) )
+	spacenav? ( dev-libs/libspnav )
+	sql? ( dev-qt/qtsql:5 )
+	truetype? ( media-libs/freetype:2 )
+	webkit? ( dev-qt/qtwebkit:5 )
+"
+RDEPEND="
+	${COMMON_DEPEND}
+	$(add_frameworks_dep khtml)
+	$(add_frameworks_dep kjobwidgets)
+	$(add_frameworks_dep knotifications)
+	$(add_frameworks_dep knotifyconfig)
+	dev-qt/qtnetwork:5
+	dev-qt/qtconcurrent:5
+	dev-qt/qtx11extras:5
+	media-libs/glew
+	media-libs/glu
+	media-libs/ilmbase
+	media-libs/libjpeg-turbo
+	media-libs/tiff
+	sci-libs/fftw
+	x11-libs/libX11
+	x11-libs/libXi
+"
+# declarative and kactivities are optional though
+DEPEND="
+	${COMMON_DEPEND}
+	$(add_frameworks_dep kdeclarative)
+	$(add_frameworks_dep kemoticons)
+	$(add_frameworks_dep kitemmodels)
+	$(add_frameworks_dep kross)
+	$(add_frameworks_dep kparts)
+	$(add_frameworks_dep threadweaver)
+	dev-lang/perl
+	dev-libs/libical
+	>=x11-misc/shared-mime-info-0.18
+	eigen? ( dev-cpp/eigen:3 )
+	iconv? ( virtual/libiconv )
+	icu? ( dev-libs/icu:= )
+	kdepim? (
+		$(add_kdeapps_dep kcalcore)
+		$(add_kdeapps_dep kcontacts)
+		$(add_kdeapps_dep kdepimlibs)
+	)
+	vc? ( dev-libs/vc )
+"
+
+#[[ ${PV} == 9999 ]] && LANGVERSION="2.4" || LANGVERSION="$(get_version_component_range 1-2)"
+#PDEPEND=">=app-office/calligra-l10n-${LANGVERSION}"
+
+RESTRICT=test
+# bug 394273
+
+pkg_pretend() {
+	kde5_pkg_pretend
+	check-reqs_pkg_pretend
+}
+
+pkg_setup() {
+	kde5_pkg_setup
+	check-reqs_pkg_setup
+}
+
+__cmake_remove_optional_module() {
+	sed -i -e "s/\(find_package($1 [0-9\.]* COMPONENTS .*\)\($2\)\([ )].*)\)/\1\3/g" CMakeLists.txt || die "couldn't remove $2 from find_package($1)"
+}
+
+src_prepare() {
+	# remove webkit from optional components
+	use dbus || __cmake_remove_optional_module Qt5 DBus
+	if ! use webkit; then
+		__cmake_remove_optional_module Qt5 WebKit
+		__cmake_remove_optional_module Qt5 WebKitWidgets
+	fi
+	if ! use sql; then
+		__cmake_remove_optional_module Qt5 Sql
+	fi
+	kde5_src_prepare
+}
+
+src_configure() {
+	mycmakeargs=(
+		-DPACKAGERS_BUILD=off
+		$(cmake-utils_use_find_package colorio OCIO)
+		$(cmake-utils_use_find_package crypt Qca-qt5)
+		$(cmake-utils_use_find_package diagrams KGantt)
+		-DCMAKE_DISABLE_FIND_PACKAGE_KChart=ON
+#		$(cmake-utils_use_find_package diagrams KChart)
```

Comment:
Why is KChart enabled by default? If it can't be disabled move the statement below PACKAGERS_BUILD and remove commented configure switch, otherwise re-enable the configure switch.